src/Orangebeard.Client/OrangebeardV2Client.cs

```diff
@@ -0,0 +1,257 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Orangebeard.Client.Entities;
+using Orangebeard.Client.OrangebeardProperties;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orangebeard.Client
+{
+    public class OrangebeardV2Client : AbstractClient
+    {
+        //private readonly ILogger<OrangebeardV2Client> LOGGER;
```

Comment:
remove?

Reply:
This was copied from the Java listener. There are some things that we want to be able to log.
Unfortunately, .NET Framework 4.6.2. doesn't have any properly documented logging framework of its own.
So, I've resolved to using log4net , the .NET sibling to log4j.